deployment/grid/terraform/compute_plane/outputs.tf

```diff
@@ -1,40 +1,74 @@
+}
 
 output "cluster_endpoint" {
   description = "Endpoint for EKS control plane."
-  value       = module.eks.eks_cluster_endpoint
+  value       = module.eks.cluster_endpoint
+}
+
+output "oidc_provider_arn" {
+  description = "OIDC Provider ARN for the EKS Cluster"
+  value       = module.eks.oidc_provider_arn
 }
 
 output "certificate_authority" {
-  description = "Endpoint for EKS control plane."
-  value       = data.aws_eks_cluster.cluster.certificate_authority
+  description = "Base64 encoded certificate data required to communicate with the cluster"
+  value       = module.eks.cluster_certificate_authority_data
+}
+
+output "eks_managed_node_groups" {
+  description = "Map of EKS Managed Node Group outputs"
+  value       = try(module.eks.eks_managed_node_groups, {})
+}
+
+output "eks_managed_node_groups_autoscaling_group_names" {
+  description = "List of EKS Managed Node Groups names"
+  value       = try(module.eks.eks_managed_node_groups_autoscaling_group_names, [])
 }
 
-output "token" {
-  description = "authentication token for EKS"
-  value       = data.aws_eks_cluster_auth.cluster.token
-  sensitive   = true
+output "self_managed_node_groups" {
+  description = "Map of self-managed node group outputs"
+  value       = try(module.eks.self_managed_node_groups, {})
+}
+
+output "self_managed_node_groups_autoscaling_group_names" {
+  description = "List of self-managed node groups names"
+  value       = try(module.eks.self_managed_node_groups_autoscaling_group_names, [])
 }
 
 output "nlb_influxdb" {
   description = "url of the NLB in front of the influx DB"
-  value = try(data.kubernetes_service.influxdb_load_balancer.status.0.load_balancer.0.ingress.0.hostname,"google.com")
+  value       = data.kubernetes_service_v1.influxdb_load_balancer.status.0.load_balancer.0.ingress.0.hostname
 }
 
 output "cognito_userpool_arn" {
-  description = "url of the NLB in front of the influx DB"
-  value = aws_cognito_user_pool.htc_pool.arn
+  description = "Cognito User Pool ARN"
+  value       = aws_cognito_user_pool.htc_pool.arn
 }
 
 output "cognito_userpool_id" {
-  description = "url of the NLB in front of the influx DB"
-  value = aws_cognito_user_pool.htc_pool.id
+  description = "Cognito User Pool ID"
+  value       = aws_cognito_user_pool.htc_pool.id
 }
 
 output "cognito_userpool_client_id" {
-  description = "url of the NLB in front of the influx DB"
-  value = aws_cognito_user_pool_client.user_data_client.id
-}
\ No newline at end of file
+  description = "Cognito User Pool Client ID"
+  value       = aws_cognito_user_pool_client.user_data_client.id
+}
+
+output "agent_permissions_policy_arn" {
+  description = "OIDC Provider ARN for the EKS Cluster"
+  value       = aws_iam_policy.agent_permissions.arn
+}
+
+output "grafana_ingress_domain" {
+  description = "Ingress Domain for Grafana"
+  value       = "https://${data.kubernetes_ingress_v1.grafana_ingress.status.0.load_balancer.0.ingress.0.hostname}"
```

Comment:
Shouldn't we use `try` as well for this output ?

Reply:
Nice catch as well! We shouldn't need nor use the try as we want things to fail if the value of the data source isnt correct. Thus, its the other `try` options that needs to be cleaned up rather, when using this data source.